source/midi_stats.py

```diff
@@ -0,0 +1,500 @@
+from joblib import Parallel, delayed
+from pathlib import Path
+import pandas as pd
+from pretty_midi import program_to_instrument_name, PrettyMIDI
+
+from utils import compute_list_average
+
+# TODO : implement util function get_files (after merging with master)
+# TODO : replace categorize_midi_instrument (after merging with master)
+# TODO : add enrichment
+# TODO : include types
+
+
+def categorize_midi_instrument(program_number):
+    if 0 <= program_number <= 7:
+        return "Piano"
+    elif 8 <= program_number <= 15:
+        return "Chromatic Percussion"
+    elif 16 <= program_number <= 23:
+        return "Organ"
+    elif 24 <= program_number <= 31:
+        return "Guitar"
+    elif 32 <= program_number <= 39:
+        return "Bass"
+    elif 40 <= program_number <= 47:
+        return "Strings"
+    elif 48 <= program_number <= 55:
+        return "Ensemble"
+    elif 56 <= program_number <= 63:
+        return "Brass"
+    elif 64 <= program_number <= 71:
+        return "Reed"
+    elif 72 <= program_number <= 79:
+        return "Pipe"
+    elif 80 <= program_number <= 87:
+        return "Synth Lead"
+    elif 88 <= program_number <= 95:
+        return "Synth Pad"
+    elif 96 <= program_number <= 103:
+        return "Synth Effects"
+    elif 104 <= program_number <= 111:
+        return "Ethnic"
+    elif 112 <= program_number <= 119:
+        return "Percussive"
+    elif 120 <= program_number <= 127:
+        return "Sound Effects"
```

Comment:
Isn't that redundant with the familiarizing script? 
Maybe it could be merged for consistency?